L1Trigger/TrackFindingTMTT/data/README

```diff
@@ -0,0 +1 @@
+Add a symbolic link from here to the MCsamples/ directory that can be found in https://github.com/cms-data/L1Trigger-TrackFindingTMTT , where .txt files listing available MC samples can be found.
```

Comment:
The contents of https://github.com/cms-data/L1Trigger-TrackFindingTMTT will be automatically linked to L1Trigger/TrackFindingTMTT/data by the CMS build system, once all the PRs are integrated. Therefore, I don't think this note is necessary.

Reply:
@tomalin I still think this can be removed.